### PR TITLE
Update SRG-APP-000023-CTR-000055 for OCP4 STIG

### PR DESCRIPTION
**Pull request by DhritiShikhar**
#### Description:

- Added `kubeadmin_removed` automated check
- Added `ocp_no_ldap_insecure` check which is related rule
- Added a check and fixtext.

#### Rationale:

- OCP4 STIG content

### CODE REVIEW

File: controls/srg_ctr/SRG-APP-000023-CTR-000055.yml [hunk 1]
```diff
@@ -4,7 +4,107 @@ controls:
   - medium
   title: {{{ full_name }}} must use a centralized user management solution to
     support account management functions.
-  rules:
+  related_rules:
```
Comment (jhrozek) [inline controls/srg_ctr/SRG-APP-000023-CTR-000055.yml:7]: Why did you move these rules to related? What is in your opinion missing from their checks that this control requies? btw maybe you were looking at an outdated version of the spreadsheet, these days we do allow anything by htpasswd

File: controls/srg_ctr/SRG-APP-000023-CTR-000055.yml [hunk 2]
```diff
@@ -4,7 +4,107 @@ controls:
...
   - idp_is_configured
   - ocp_idp_no_htpasswd
-  status: automated
+  status: manual
+  check: |-
+    Verify that the authentication operator is configure to use either an LDAP or a OpenIDConnect provider:
+
+    > oc get oauth cluster -o jsonpath="{.spec.identityProviders[*].type}{'\n'}"
+
+    If the output lists any other type besides LDAP or OpenID, this is a finding.
```
Comment (jhrozek) [inline controls/srg_ctr/SRG-APP-000023-CTR-000055.yml]: anything but htpasswd should be fine.

**Review by jhrozek — COMMENTED** [review]
two questions inline

File: controls/srg_ctr/SRG-APP-000023-CTR-000055.yml (hunk 1 above)
Comment (DhritiShikhar) [reply controls/srg_ctr/SRG-APP-000023-CTR-000055.yml]: I added one rule to related which is `ocp_no_ldap_insecure`

File: controls/srg_ctr/SRG-APP-000023-CTR-000055.yml (hunk 2 above)
Comment (DhritiShikhar) [reply controls/srg_ctr/SRG-APP-000023-CTR-000055.yml]: Updated the text.

Comment (jhrozek) [reply controls/srg_ctr/SRG-APP-000023-CTR-000055.yml]: thanks!

File: controls/srg_ctr/SRG-APP-000023-CTR-000055.yml (hunk 1 above)
Comment (jhrozek) [reply controls/srg_ctr/SRG-APP-000023-CTR-000055.yml]: oh and the rule should reference the control, too.

Comment (DhritiShikhar) [reply controls/srg_ctr/SRG-APP-000023-CTR-000055.yml]: Addressed.
